src/lib/mdns/ServiceNaming.h

```diff
@@ -27,6 +27,18 @@
 
 namespace chip {
 namespace Mdns {
+constexpr size_t kMaxSubtypeDescSize        = 8; // max 5 decimal digits + _X prefix. + null character
+constexpr char kSubtypeServiceNamePart[]    = "_sub";
+constexpr char kCommissionableServiceName[] = "_chipc";
+constexpr char kOperationalServiceName[]    = "_chip";
+constexpr char kCommissionerServiceName[]   = "_chipd";
+constexpr char kOperationalProtocol[]       = "_tcp";
+constexpr char kCommissionProtocol[]        = "_udp";
+constexpr char kLocalDomain[]               = "local";
+
+// each includes space for a null terminator, which becomes a . when the names are appended.
+constexpr size_t kMaxCommisisonableServiceNameSize =
+    kMaxSubtypeDescSize + sizeof(kSubtypeServiceNamePart) + sizeof(kCommissionableServiceName);
```

Comment:
These include null terminators so it will be bigger than expected.

Reply:
See comment - each includes a null terminator, but that needs to be converted to a . when the name is created.